src/frontend/main.go

```diff
@@ -215,6 +207,18 @@ func initTracing(log logrus.FieldLogger) {
 	log.Warn("could not initialize stackdriver exporter after retrying, giving up")
 }
 
+func initTracing(log logrus.FieldLogger) {
+	// This is a demo app with low QPS. trace.AlwaysSample() is used here
+	// to make sure traces are available for observation and analysis.
+	// In a production environment or high QPS setup please use
+	// trace.ProbabilitySampler set at the desired probability.
+	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
```

Comment:
again, should we put the sampling rate to the service where requests don't start from?

Reply:
yeah, requests originate from frontend. so I'm not sure if having this here does anything in particular.

Reply:
my bad, this is the frontend code I'm reading.